src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={classNames(
+                { 'is-active': tab.id === selectedTabId },
+              )}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => onTabSelected(tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="tab-content">
+        {tabs[tabs.findIndex(tab => tab.id === selectedTabId)].content}
+      </div>
```

Comment:
You don't need to find selectedTab, because you already have it. Think about how to rewrite it